utils/brush/proc_macros/internal.rs

```diff
@@ -220,3 +327,29 @@ pub(crate) fn extract_attr(attrs: &mut Vec<syn::Attribute>, ident: &str) -> Vec<
 pub(crate) fn new_attribute(attr_stream: TokenStream2) -> syn::Attribute {
     syn::parse2::<Attributes>(attr_stream).unwrap().attr()[0].clone()
 }
+
+/// Computes the BLAKE-2b 256-bit hash for the given input and stores it in output.
+#[inline]
+pub fn blake2b_256(input: &[u8], output: &mut [u8]) {
```

Comment:
Why can't we use Blake2 from ink_env::hash?

Reply:
Because it will be evaluated during compilation time instead of runtime